src/commands/importWebApp/importWebApp.ts

```diff
@@ -0,0 +1,321 @@
+        const existingOperationNames = getAllOperationNames(allOperations);
+        const existingOperationDisplayNames = getAllOperationDisplayNames(allOperations);
+        HttpMethods.forEach(method => {
+            let operationId = getBsonObjectId();
+            let operationDisName = `${apiName}_${method}`;
+            if (existingOperationNames.includes(operationId) || existingOperationDisplayNames.includes(operationDisName)) {
+                ext.outputChannel.appendLine(localize("importWebApp", "Resolving conflict operations..."));
+                const subfix = genUniqueOperationNameSubfix(operationId, operationDisName, existingOperationNames, existingOperationDisplayNames);
+                operationId = `${operationId}-${subfix}`;
+                operationDisName = `${operationDisName}-${subfix}`;
+            }
+            const operation = getNewOperation(apiId, method, operationId, operationDisName);
+            operations.push(operation);
+        });
+    } else {
+        HttpMethods.forEach(method => {
+            const operationId = getBsonObjectId();
+            const operation = getNewOperation(apiId, method, operationId, `${apiName}_${method}`);
+            operations.push(operation);
+        });
+    }
+    return operations;
+}
+
+function getNewOperation(apiId: string, method: string, operationId: string, displayName: string): OperationContract {
+    return {
+        id: `${apiId}/operations/${operationId}`,
+        name: operationId,
+        displayName: displayName,
+        method: method,
+        description: "",
+        urlTemplate: "*",
+        templateParameters: []
+    };
+}
+
+function getBsonObjectId(): string {
+    // tslint:disable: no-bitwise
+    const timestamp = (new Date().getTime() / 1000 | 0).toString(16);
+
+    return timestamp + "xxxxxxxxxxxxxxxx".replace(/[x]/g, getRandomHex()).toLowerCase();
+}
+
+function getRandomHex(): string {
+    // tslint:disable-next-line: insecure-random
+    return (Math.random() * 16 | 0).toString(16);
+}
+
+// tslint:disable: no-any
+async function parseDocument(documentJson: any): Promise<IOpenApiImportObject> {
+    try {
+        // tslint:disable-next-line: no-unsafe-any
+        return await new OpenApiParser().parse(documentJson);
+    } catch (error) {
+        throw new Error(processError(error, localize("openApiJsonParseError", "Could not parse the provided OpenAPI document.")));
+    }
+}
+
+function genUniqueOperationNameSubfix(opName: string, opDisName: string, existingOperationNames: string[], existingOperationDisplayNames: string[]): number {
+    let cnt = 0;
+    let curName = opName;
+    while (existingOperationNames.includes(curName) || existingOperationDisplayNames.includes(opDisName)) {
+        cnt++;
+        curName = opName.concat("-", cnt.toString());
+    }
+    return cnt;
+}
+
+function getAllOperationNames(operations: OperationCollection): string[] {
```

Comment:
avoid looping twice through the operations. getAllOperationNames and getAllOperationDisplayNames

combine and return a dictionary <name, displayname>